ci/activemq.rb

```diff
@@ -0,0 +1,72 @@
+require './ci/common'
+
+def activemq_version
+  ENV['FLAVOR_VERSION'] || '5.11.1'
+end
+
+def activemq_rootdir
+  "#{ENV['INTEGRATIONS_DIR']}/activemq_#{activemq_version}"
+end
+
+namespace :ci do
+  namespace :activemq do |flavor|
+    task :before_install => ['ci:common:before_install']
+
+    task :install => ['ci:common:install'] do
+      unless Dir.exist? File.expand_path(activemq_rootdir)
+        sh %(curl -s -L\
+             -o $VOLATILE_DIR/activemq-#{activemq_version}.tar.gz\
+             http://archive.apache.org/dist/activemq/#{activemq_version}/apache-activemq-#{activemq_version}-bin.tar.gz)
+        sh %(mkdir -p #{activemq_rootdir})
+        sh %(tar zxf $VOLATILE_DIR/activemq-#{activemq_version}.tar.gz\
+             -C #{activemq_rootdir} --strip-components=1)
+      end
+    end
+
+    task :before_script => ['ci:common:before_script'] do
+      sh %(tar zxf $TRAVIS_BUILD_DIR/ci/resources/activemq/kahadb.tar.gz\
+           -C #{activemq_rootdir}/data)
```

Comment:
Since this directory is always recreated, could you remove it in `before_cache` ? (otherwise the cache will be reuploaded everytime)